src/processing/dom/html-namespaces.js

```diff
@@ -0,0 +1,13 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
+export default {
+    HTML:   'http://www.w3.org/1999/xhtml',
+    MATHML: 'http://www.w3.org/1998/Math/MathML',
+    SVG:    'http://www.w3.org/2000/svg',
+    XLINK:  'http://www.w3.org/1999/xlink',
+    XML:    'http://www.w3.org/XML/1998/namespace',
+    XMLNS:  'http://www.w3.org/2000/xmlns/'
+};
```

Comment:
> But, I have no idea why we need ALL namespace constants - only two of them used in code.

For future use.
For example, for https://github.com/DevExpress/testcafe-hammerhead/issues/477  issue we will need the XML namespace.

> I don't see why we need dedicated file for them. It's just fine to have inline constants in each file in that case.

The same logical costants (namespaces) grouped in the separate file.